inc/composer/class-docker-compose-generator.php

```diff
@@ -129,16 +129,16 @@ protected function get_php_reusable() : array {
 				'ELASTICSEARCH_HOST' => 'elasticsearch',
 				'ELASTICSEARCH_PORT' => 9200,
 				'AWS_XRAY_DAEMON_HOST' => 'xray',
-				'S3_UPLOADS_ENDPOINT' => "https://{$this->tld}/",
+				'S3_UPLOADS_ENDPOINT' => 'http' . ( $this->args['secure'] ? 's' : '' ) . "://s3-{$this->hostname}.localhost/",
```

Comment:
For readability lets add a helper method to read the `secure` arg and modify the URL e.g.:

```php
$this->set_url_scheme( "http://s3-{$this->hostname}.localhost/" )
```